docs/model-dev-guide/api-guides/apis-howto/api-keras-ug.rst

```diff
@@ -7,129 +7,155 @@
+the Determined cluster.  It reports train and test metrics, it reports progress, it saves
+checkpoints, and it uploads them to checkpoint storage.  It also handles preemption signals from the
+Determined master (such as if you pause your experiment), shutting down training, then it restores
+training from where it left off when the experiment continues.
 
-TensorFlow Keras trials are checkpointed to a file named ``determined-keras-model.h5`` using
-``tf.keras.models.save_model``. You can learn more from the `TF Keras docs
-<https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/keras/models/save_model>`__.
+The ``DeterminedCallback`` has only three required inputs:
+   -  the ``core_context`` you already created
+   -  a ``checkpoint`` UUID to start training from, or ``None``.
+   -  a ``continue_id`` used to decide how to treat the checkpoint.
 
-***********
- Callbacks
-***********
+In training jobs, a value for ``checkpoint`` should be obtained from
+``det.get_cluster_info().latest_checkpoint``, which will automatically be populated with the latest
+checkpoint saved by this trial, if there is one.
 
-To execute arbitrary Python code during the lifecycle of a :class:`~determined.keras.TFKerasTrial`,
-implement the :class:`determined.keras.callbacks.Callback` interface (an extension of the
-``tf.keras.callbacks.Callbacks`` interface) and supply them to the
-:class:`~determined.keras.TFKerasTrial` by implementing
-:meth:`~determined.keras.TFKerasTrial.keras_callbacks`.
+The ``continue_id`` helps the ``DeterminedCallback`` decide if the provided checkpoint represents
+just the starting weights and training should begin at epoch=0, or if the checkpoint represents a
+partially complete training that should pick up where it left off (at epoch > 0).  The provided
+``continue_id`` is saved along with every checkpoint, and when loading the starting checkpoint, if
+the ``continue_id`` matches what was in the checkpoint, training state is also loaded from the
+checkpoint.  In training jobs, an easy value for ``continue_id`` is
+``det.get_cluster_info.trial.trial_id``.
 
-.. _keras-profiler:
+See the reference for :class:`~determined.keras.DeterminedCallback` to see the optional parameters.
+
+.. code:: python
+
+   info = det.get_cluster_info()
+   assert info and info.task_type == "TRIAL", "this example only runs as a trial on the cluster"
+
+   det_cb = det.keras.DeterminedCallback(
+       core_context,
+       checkpoint=info.latest_checkpoint,
+       continue_id=info.trial.trial_id,
+    )
 
 ***********
- Profiling
+ Load Data
 ***********
 
-Determined supports integration with the native TF Keras profiler. Results will automatically be
-uploaded to the trial's TensorBoard path and can be viewed in the Determined Web UI.
+Loading data is done normally, though additional considerations may arise if your existing data
+loading code is not already container-ready.  See :ref:`load-model-data` for more information.
```

Comment:
Loading data is done as usual, though additional considerations may arise if your existing data-loading code is not container-ready. For more details, see :ref:`load-model-data`.